docs_src/microservices/application/Configuration.md

```diff
@@ -0,0 +1,74 @@
+# Application Services Configuration
+
+This section describes the configuration elements that are unique to Application Services
+
+Please refer to the general [Configuration documentation](../../configuration/Ch-Configuration/#configuration) for configuration properties common across all services.
+
+!!! note
+    `*`indicates the configuration value can be changed on the fly if using a configuration provider (like Consul).
+
+!!! note
+    `**`indicates the configuration value can be changed but the service must be restarted.
+
+|Configuration  |     Default Value     | Description |
```

Comment:
I think it makes sense to create a separate table per section -- its a bit difficult to track which section is separate.

Reply:
Maybe actually break out the "writable section' for asc out into its own subsection under configurable and then leave this here as configuration.

Reply:
K, agreed, maybe even level out 

> I think it makes sense to create a separate table per section -- its a bit difficult to track which section is separate.

Agreed.

Reply:
> Maybe actually break out the "writable section' for asc out into its own subsection under configurable and then leave this here as configuration.

K, that separate configuration section for ASC. :-)

Reply:
separate configuration section for ASC will be in it's own PR